templates/ep19/bs/cart/step_1_choose_type_of_order.html

```diff
@@ -20,6 +20,8 @@ <h3>Conference and Training tickets:</h3>
             <div class='col-md-4'>
                 <p class='cart-step1-description'>
                 For people enjoying Python from home.
+                If you are programming Python in your free time or as a freelancer
```

Comment:
I think listing freelancers in this category is going to lead to confusion as most probably do need an invoice - I think saying something along the lines of  `If you are programming in Python in your free time or self-funding the tickets and don't need a VAT invoice` would be clearer. Not considering this a blocker though.

Reply:
This is to clarify the possibility for freelancers to use personal tickets as well - provided they don't need a VAT invoice. For some this may be attractive, since the personal ticket is much cheaper than the business one, even when considering that you can't get the VAT back.